src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -152,12 +153,16 @@ IObservable<Unit> OnCloneRepository(object state)
                     notificationService.ShowError(Resources.RepositoryCloneFailedNoSelectedRepo);
                     return Observable.Return(Unit.Default);
                 }
-                
+
                 // The following is a noop if the directory already exists.
                 operatingSystem.Directory.CreateDirectory(BaseRepositoryPath);
 
                 return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath)
-                    .Do(_ => this.usageTracker.IncrementCloneCount());
```

Comment:
Not right now, but one of my alternate implementations used a custom clone service using libgit2sharp directly instead of the TeamExplorer service, overriding the default service just for the start page cloning, and that one signaled multiple times :-P